imports/api/meters/actions.js

```diff
@@ -130,6 +130,14 @@ Meters.batchActions = {
 
 //-----------------------------------------------
 
+function isTooLargeValue(doc) {
+  const meter = Meters.findOne(doc._id);
+  const lastReadingValue = meter.lastReading()?.value;
+  const lastDifference = meter.lastBilling()?.value || lastReadingValue - meter.readings[meter.readings.length - 2]?.value;
+  if (lastDifference > 0 && (doc.reading.value - lastReadingValue) > (lastDifference * 10)) return true;
```

Comment:
Ha e helyett  azt írnánk, hogy doc.reading.value > meter.getEstimatedValue() * 10, az nem jobb olyan szempontból, hogy figyelembe veszi az eltelt időt is. Mert ha az utóbbi 2 leolvasás kozott 1 hónap telt el, de a mostani elott 12 hónap, akkor jogos lehet hogy 10-szer nagyobb a difference.